test/FunctionalTestUtils/project.json

```diff
@@ -1,18 +1,19 @@
 {
+  "version": "1.0.1",
```

Comment:
These are tests and it just defaulted to 1.0.0, I figured I would explicitly add it and set it to 1.0.1.
The tests don't go in the NuGet package.